.travis.yml

```diff
@@ -295,6 +295,9 @@ jobs:
     - os: linux
       arch: arm64
       env: CC=gcc CXX=g++ CONFIGURE_FLAGS="--with-lg-page=16" EXTRA_CFLAGS="-Werror -Wno-array-bounds"
+    - os: linux
+      arch: arm64
+      env: CC=gcc CXX=g++ CONFIGURE_FLAGS="--with-lg-page=16 --with-lg-hugepage-29" EXTRA_CFLAGS="-Werror -Wno-array-bounds"
```

Comment:
Probably should be `--with-lg-hugepage=29` instead of `--with-lg-hugepage-29`. Here and below in few places.

Reply:
Thanks, made a typo in the script